docs/book/aya/crosscompile.md

```diff
@@ -0,0 +1,49 @@
+# Cross-compiling aya-based programs
+
+The instructions below show how to cross compile aya eBPF programs on Macs.
+Cross compiling on other systems is possible too and we're going
+to add instructions to do that soon (PRs welcome!).
+
+# Cross-compiling aya-based programs on Mac
+
+Cross compilation should work on both Intel and Apple Silicon Macs.
+
+1. Install `rustup` following the instructions on <https://rustup.rs/>
+1. Install a rust stable and nightly toolchains:
+```bash
+rustup install stable
+rustup toolchain install nightly --component rust-src
+```
+1. Install the [rustup target](https://doc.rust-lang.org/nightly/rustc/platform-support.html#tier-1-with-host-tools) for your Linux target platform:
+```bash
+ARCH=x86_64
```

Comment:
You need to export this or it won't work

Reply:
why? this is just for the textual substitution for `"x86_64"` or `"aarch64"` in the commands below which would be executed in the same terminal/shell session - _not_ used inside of the called scripts or programs.  Tested on a "fresh" Mac by copying from the [preview](https://deploy-preview-132--aya-rs.netlify.app/book/aya/crosscompile/) site - worked fine for me.

Reply:
oh yep you're right!